code/style.css

```diff
@@ -1,7 +1,497 @@
+  font-family: 'Roboto Slab', serif; 
+  font-size: 16px;
+}
+
+.smaller {
+  font-size: 14px;
+  color: rgba(255,255,255,0.7);
+}
+
+.smaller a {
+  font-family: 'Roboto Slab', serif; 
+  color: #5DC3CA;
+  text-transform: none;
+  font-size: 14px;
+}
+
+/* ===== form input styling ===== */
+
+label {
+  margin: 0 0 8px 16px;
+}
+
+.bold {
+  font-weight: 700;
+}
+
+input[type=text], input[type=email] {
+  border-radius: 24px;
+  border: 1px solid rgba(0,0,0,0.20);
+  padding: 10px 24px;
+
+  font-family: 'Roboto Slab', serif; 
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  margin-bottom: 24px;
 }
+
+input[type=text]:focus, input[type=email]:focus {
+  border: 1px solid #BF85BF;
+  outline-width: 0;
+  box-sizing: border-box;
+  box-shadow: inset 0 0 0px 2px #BF85BF;
+}
+
+.slider {
+  -webkit-appearance: none;
+  appearance: none;
+  height: 8px;
+  background-color: rgba(0,0,0,0.11);
+  border-radius: 100px;
+  width: 80%;
+}
+
+.slider:focus {
+  outline-width: 0;
+}
+
+.form-checkcontainer, .form-slidecontainer {
+  width: 100%;
+  padding: 0 16px 0 16px;
+  margin-bottom: 24px;
+}
+
+.form-checkcontainer p {
+  margin-top: 0;
+}
+
+input[type="checkbox"] {
+  margin-bottom: 16px;
+}
+
+.form-slidecontainer {
+  display: flex;
+  flex-direction: column;
+}
+
+/* SLIDER CODE  */
+
+.sliderbox {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.slider::-webkit-slider-thumb {
+  width: 20px; /* Set a specific slider handle width */
+  height: 20px; /* Slider handle height */
+  background: #BF85BF;
+  border-radius: 100%;
+  -webkit-appearance: none; /* Override default look */
+  appearance: none;
+}
+
+.slider::-moz-range-thumb {
+  width: 20px; /* Set a specific slider handle width */
+  height: 20px; /* Slider handle height */
+  background: #BF85BF;
+  border-radius: 100%;
+}
+
+.sliderbox > p {
+  margin-left: 16px;
+}
+
+/* CHECKBOX CODE  */
+
+.checkgroup {
+  display: flex;
+  flex-direction: row;
+  justify-content: left;
+}
+
+.checkbox {
+  appearance: none;
+  border: 1px solid rgba(0,0,0,0.3);
+  width: 18px;
+  height: 18px;
+  margin: 0px;
+  border-radius: 4px;
+  outline-width: 0;
+}
+
+.checkbox:active, .checkbox:checked:active {
+  background: #D39FD3;
+  border: 1px solid #D39FD3;
+  box-shadow: inset 0px 0px 0px 2px #FFFFFF;
+  outline-width: 0;
+}
+
+.checkbox:checked {
+  background: #BF85BF;
+  border: 1px solid #BF85BF;
+  box-shadow: inset 0px 0px 0px 2px #FFFFFF;
+  outline-width: 0;
+}
+
+
+.form {
+  display: flex;
+  flex-direction: column;
+}
+
+.productInfo {
+  background-color: #182025;
+  min-height: 350px;
+  padding: 40px 0 24px 0;
+}
+
+.productInfo-content {
+  height: 100%;
+  max-width: 1200px;
+  margin: 0 auto;
+  display: flex;
+}
+
+.hibr-info, .hibr-image {
+  height: 0;
+}
+
+.hibr-info {
+  flex-grow: 5;
+  min-width: 50%;
+}
+
+.hibr-image img {
+  position: relative;
+  max-width: 670px;
+  top: -54px;
```

Comment:
Hmm, wasn't there a batter idea to achieve same result? First thing that come to my head are paddings and margins